crates/goose-mcp/src/computercontroller/platform/macos.rs

```diff
@@ -0,0 +1,25 @@
+use super::SystemAutomation;
+use std::path::PathBuf;
+use std::process::Command;
+
+pub struct MacOSAutomation;
+
+// MacOSAutomation is Send + Sync because it contains no shared state
+unsafe impl Send for MacOSAutomation {}
+unsafe impl Sync for MacOSAutomation {}
```

Comment:
is it necessary to manually `unsafe impl Send` here? i think an empty struct would automatically derive `Send + Sync` 

if there's a need for it can you add a comment explaining why?

Reply:
I am not great at ruby, so probably not. I just made it work lol